.ebextensions/09_extract_local_configuration.config

```diff
@@ -0,0 +1,3 @@
+container_commands:
+  01_extract_local_configuration:
+    command: "unzip -p /tmp/local_configurations.zip ${EB_ENVIRONMENT}.json > config/local-production.json"
```

Comment:
The `EB_ENVIRONMENT` variable is one that we created ourselves in the console so that a given environment (e.g., `suitec-preview-service-dev`) knows its own name. This variable is accessible from `container_commands` but not `files`; so we download all files as a zip and then extract the environment-appropriate file in a container command.